packages/fxa-settings/src/pages/Signin/SigninTokenCode/index.tsx

```diff
@@ -101,25 +98,23 @@ const SigninTokenCode = ({
     setResendCodeLoading(true);
     try {
       await session.sendVerificationCode();
-      if (banner.type === BannerType.success) {
+      if (showResendSuccessBanner) {
+        // shake the banner if it is already displayed
         setAnimateBanner(true);
       } else {
-        setBanner({
-          type: BannerType.success,
-        });
+        setShowResendSuccessBanner(true);
       }
     } catch (error) {
-      const localizedErrorMessage =
+      setShowResendSuccessBanner(false);
+      // TODO in FXA-7914 - verify if we only want to display a specific message for throttled errors
```

Comment:
Should we check if the error is an incorrect/invalid code and if so display that in a tooltip (to also return focus to the input with error styling), and display every other error in a banner (vs only displaying the throttling errors in a banner)?